app/view/forms/submission_form.py

```diff
@@ -90,6 +87,9 @@ def update_project(self, data):
 
             if previous_submission:
                 self.submission.studyDesign = previous_submission.studyDesign
+                # Clear out experiment ids:
+                for experiment_data in self.submission.studyDesign.get('experiments', []):
+                    experiment_data['publicId'] = None
```

Comment:
When copying a previous submission, we want to be careful not to copy the experiment ids as well.